pwa-devdocs/src/technologies/basic-concepts/internationalization/index.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Internationalization Framework
+---
+
+Internationalization (i18n) is a feature that lets you localize content for a culture, region, or language.
+This feature is often associated with localization (l10n), which is the process of transforming content for a specific locale.
+
+## Internationalization in core Magento versus PWA Studio
+
+The core Magento application includes an i18n feature that provides translated text to the frontend theme.
+This feature uses dictionary files inside language packages to provide translation data for Magento when it renders a page.
+The language packages themselves are Magento extensions the application installs using Composer.
+
+For more information, see the Magento core topic: [Translations overview][].
+
+The tight coupling between core Magento's i18n feature and its frontend theme makes it difficult to use the same translation mechanisms in PWA Studio storefronts.
+Instead, PWA Studio provides its own i18n framework that follows a similar design as the one in core Magento.
+
+## How it works
+
+PWA Studio provides a context provider for translations called the [`LocaleProvider`][].
```

Comment:
I would highlight this `LocaleProvider` just wraps the `react-intl` `IntlProvider` and just provides a mechanism for wiring everything up to make localization a success. Speaking about `__fetchLocaleData__` and linking to more information would be helpful.

Reply:
I see you do this below, but I think it's crucial to not suggest we invented something new.

Reply:
content revised to reflect this